test/3x/torch/quantization/test_pt2e_quant.py

```diff
@@ -114,6 +114,18 @@ def calib_fn(model):
         config.freezing = True
         q_model_out = q_model(*example_inputs)
         assert torch.allclose(float_model_output, q_model_out, atol=1e-2), "Quantization failed!"
+
+        # test save and load
+        q_model.save(
+            example_inputs=example_inputs,
+            output_dir="./saved_results",
+        )
+        from neural_compressor.torch.quantization import load
+
+        loaded_quantized_model = load("./saved_results")
+        loaded_q_model_out = loaded_quantized_model(*example_inputs)
+        assert torch.allclose(loaded_q_model_out, q_model_out)

```

Comment:
nit: 1) use `torch.equal` instead of `torch.allcose`; 2) clean the `saved_results` after executing UT.

Reply:
1)fixed https://github.com/intel/neural-compressor/pull/1918/commits/2708ed258f6d5212a5aedce97b79533914aea2fe 2) https://github.com/intel/neural-compressor/pull/1918/files#diff-52e7e2326ea80a3d6263f9cd941421a4cacab69718e8b5fb364761c19298e6adR35:~:text=def%20teardown_class(,%3DTrue)